holidayextras/holidayextras.github.com
# NPM, Grunt, LESS, Uglify & Deferred Loading

## What does this PR do?
- Adds NPM
- Converts CSS to LESS
- Creates critical CSS and deferrs all other styles
- Minify/Uglify CSS and JS

## How to test
- Checkout branch
- `npm install`
- Try the new npm scripts - `npm run less`, `npm run uglify`, `npm run watch` and `npm start`
- Run `npm start`
- Visit `http://localhost:4000/`
- Make sure the styling matches `http://hungrygeek.holidayextras.co.uk/`


## Give me a GIF
![](http://i.giphy.com/EGZkj53VzM18Y.gif)

## Review

#### Quality checklist (for PR author to complete BEFORE code review):

#### IP or Developer Review:

#### Software Engineer, Software Architect or Developer review:
